### PR TITLE
Feature/default options

### PR DESCRIPTION
**Pull request by zachdt**
- Added Mock Options to Options context, currently only pushes 1 option at the current spot price strike (estimated by 10$, might not be a great idea for most assets).
- Improved page and footer styling
- Added mock option to option table
- Hooked up expiry filter to options table

TODO: Add mock option / initialize order type

### CODE REVIEW

File: app/src/components/Market/FilterBar/FilterBar.tsx [hunk 1]
```diff
@@ -35,8 +40,8 @@ const FilterBar: React.FC<FilterBarProps> = (props) => {
             />
           </Toggle>
           <Spacer size="lg" />
-          <StyledSelect>
-            <StyledOption>November 2</StyledOption>
+          <StyledSelect value={expiry} onChange={handleFilter}>
+            <StyledOption>December 30 2020</StyledOption>
           </StyledSelect>
```
Comment (Alexangelj) [inline app/src/components/Market/FilterBar/FilterBar.tsx:45]: haha, finally!

File: app/src/components/Footer/components/Nav.tsx [hunk 2]
```diff
@@ -39,6 +39,10 @@ const Nav: React.FC = () => {
 
 const StyledNav = styled.nav`
   display: flex;
+  background-color: ${(props) => props.theme.color.grey[800]};
+  border-radius: 10px;
+  opacity: 90%;
+  pointer-events: all;
```
Comment (Alexangelj) [inline app/src/components/Footer/components/Nav.tsx:45]: alphabetize

**Review by Alexangelj — APPROVED** [review]
Looks like you did some great work here! Merge

File: app/src/components/Footer/components/Nav.tsx (hunk 2 above)
Comment (zachdt) [reply app/src/components/Footer/components/Nav.tsx:45]: i wish this was a lint feature... hmmm tool idea

**Pull request merged**